Assets/SEE/Net/Network.cs

```diff
@@ -629,153 +573,201 @@ private void OnSceneUnloaded(Scene scene)
+        private void OnClientDisconnectCallbackForServer(ulong client)
+        {
+            Debug.Log($"Client {client} has disconnected.\n");
+        }
+
+        /// The IP4 address, port, and protocol.
+        /// </summary>
+        private string ServerAddress => $"{ServerIP4Address}:{ServerPort} (UDP)";
+
+        /// <summary>
+        /// Starts a host process, i.e., a server and a local client.
         ///
         /// Note: This method starts a co-routine and then returns to the caller immediately.
         /// The <paramref name="callBack"/> tells the caller that the co-routine has come to
         /// an end.
         /// </summary>
         /// <param name="callBack">a callback to be called when done; its parameter will be true
         /// in case of success or otherwise false</param>
-        public void StartClient(CallBack callBack)
+        public void StartHost(CallBack callBack)
         {
-            StartCoroutine(ShutdownNetwork(InternalStartClient));
+            StartCoroutine(ShutdownNetwork(InternalStartHost));
 
-            void InternalStartClient()
+            void InternalStartHost()
             {
-                Debug.Log($"Client is trying to connect to server {ServerAddress}...\n");
+                Debug.Log($"Host is starting to listen at {ServerAddress}...\n");
+                Debug.Log($"Local client is trying to connect to server {ServerAddress}...\n");
                 try
                 {
-                    if (NetworkManager.Singleton.StartClient())
+                    NetworkManager.Singleton.ConnectionApprovalCallback = ApprovalCheck;
+                    if (NetworkManager.Singleton.StartHost())
                     {
-                        StartUp();
+                        InitializeGame();
+                        NetworkManager.Singleton.OnClientConnectedCallback += OnClientConnectedCallbackForServer;
+                        NetworkManager.Singleton.OnClientDisconnectCallback += OnClientDisconnectCallbackForServer;
                     }
                     else
                     {
-                        throw new NoServerConnection($"Could not connect to server {ServerAddress}");
+                        throw new CannotStartServer($"Could not start host {ServerAddress}");
                     }
                 }
                 catch (Exception exception)
                 {
-                    Debug.LogError($"Could not connect to server {ServerAddress}. Details: {exception.Message}\n");
                     callBack(false, exception.Message);
                     throw;
                 }
-
-                StartCoroutine(WaitUntilConnected());
+                callBack(true, $"Host started at {ServerAddress}.");
             }
+        }
 
-            IEnumerator WaitUntilConnected()
+        /// <summary>
+        /// Checks whether a specific client is authorised to establish a connection to the server.
```

Comment:
Consistency with rest of SEE (AE vs BE):
```suggestion
        /// Checks whether a specific client is authorized to establish a connection to the server.
```